build-tools/packages/build-tools/src/fluidBuild/fluidUtils.ts

```diff
@@ -148,7 +148,7 @@ const configExplorer = cosmiconfigSync(configName, {
  * Set to true when the default config is returned by getFluidBuildConfig so that repeated calls to the function don't
  * result in repeated searches for config.
  */
-let defaultConfigLoaded = false;
+const defaultSearchDir = new Set<string>();
```

Comment:
```suggestion
 * Contains directories previously searched which contained no explicit fluidBuild config. 
 * This allows avoiding repeated searches for config.
 */
const defaultSearchDir = new Set<string>();
```

Reply:
Not sure it's relevant here, but this isn't really a list of previously searched directories. It's rather a list of _starting directories_ for the config search, which traverses up the file tree looking for a config.

Reply:
Fixed the comment to reflect this.